src/metabase/models/database.clj

```diff
@@ -311,6 +317,10 @@
 
 (t2/define-after-update :model/Database
   [database]
+  ;; Like for `is_audit`, we do not support removing this flag:
+  (when (:is_attached_dwh database)
+    (let [admin-group (perms-group/admin)]
+      (data-perms/set-database-permission! admin-group database :perms/manage-database :no)))
```

Comment:
Some questions / things to think about.

1. When do we anticipate this property being changed? Is this related to 2?
2. What's our plan for enabling this property / stripping these permissions for existing instances? 
4. Do we not want a hook to prevent `:is_attached_dwh` being disabled?
5. Even allowing customers to enable it via the API seems like something we wouldn't want.

Reply:
Thanks, sounds solid